server/public/model/websocket_message.go

```diff
@@ -9,87 +9,98 @@ import (
-	WebsocketEventMultipleChannelsViewed              = "multiple_channels_viewed"
-	WebsocketEventPluginStatusesChanged               = "plugin_statuses_changed"
-	WebsocketEventPluginEnabled                       = "plugin_enabled"
-	WebsocketEventPluginDisabled                      = "plugin_disabled"
-	WebsocketEventRoleUpdated                         = "role_updated"
-	WebsocketEventLicenseChanged                      = "license_changed"
-	WebsocketEventConfigChanged                       = "config_changed"
-	WebsocketEventOpenDialog                          = "open_dialog"
-	WebsocketEventGuestsDeactivated                   = "guests_deactivated"
-	WebsocketEventUserActivationStatusChange          = "user_activation_status_change"
-	WebsocketEventReceivedGroup                       = "received_group"
-	WebsocketEventReceivedGroupAssociatedToTeam       = "received_group_associated_to_team"
-	WebsocketEventReceivedGroupNotAssociatedToTeam    = "received_group_not_associated_to_team"
-	WebsocketEventReceivedGroupAssociatedToChannel    = "received_group_associated_to_channel"
-	WebsocketEventReceivedGroupNotAssociatedToChannel = "received_group_not_associated_to_channel"
-	WebsocketEventGroupMemberDelete                   = "group_member_deleted"
-	WebsocketEventGroupMemberAdd                      = "group_member_add"
-	WebsocketEventSidebarCategoryCreated              = "sidebar_category_created"
-	WebsocketEventSidebarCategoryUpdated              = "sidebar_category_updated"
-	WebsocketEventSidebarCategoryDeleted              = "sidebar_category_deleted"
-	WebsocketEventSidebarCategoryOrderUpdated         = "sidebar_category_order_updated"
-	WebsocketWarnMetricStatusReceived                 = "warn_metric_status_received"
-	WebsocketWarnMetricStatusRemoved                  = "warn_metric_status_removed"
-	WebsocketEventCloudPaymentStatusUpdated           = "cloud_payment_status_updated"
-	WebsocketEventCloudSubscriptionChanged            = "cloud_subscription_changed"
-	WebsocketEventThreadUpdated                       = "thread_updated"
-	WebsocketEventThreadFollowChanged                 = "thread_follow_changed"
-	WebsocketEventThreadReadChanged                   = "thread_read_changed"
-	WebsocketFirstAdminVisitMarketplaceStatusReceived = "first_admin_visit_marketplace_status_received"
-	WebsocketEventDraftCreated                        = "draft_created"
-	WebsocketEventDraftUpdated                        = "draft_updated"
-	WebsocketEventDraftDeleted                        = "draft_deleted"
-	WebsocketEventAcknowledgementAdded                = "post_acknowledgement_added"
-	WebsocketEventAcknowledgementRemoved              = "post_acknowledgement_removed"
-	WebsocketEventPersistentNotificationTriggered     = "persistent_notification_triggered"
-	WebsocketEventHostedCustomerSignupProgressUpdated = "hosted_customer_signup_progress_updated"
+	Typing                                   WebsocketEventType = "typing"
```

Comment:
While you correctly followed the ticket, I'm still wondering if the short variable name is helpful and intent revealing enough. I'm leaning towards revering back to a slightly longer name e.g. `EventTyping` in this case.

@sudheer121 @agnivade What do you think?

Reply:
I commented here: https://github.com/mattermost/mattermost/issues/25114#issuecomment-1790002614. I'm leaning towards not changing the name at all.

Reply:
got it @agnivade, will revert to old name for `Typing` as it is clearer

Reply:
but there are many such events with short name, shouldn't we revert their names as well ?

Reply:
have reverted the `Typing` type

Reply:
Sorry, my previous comment was ambiguous. All events should keep the `WebsocketEvent` prefix.